visual-tests/tests/tab-navigation.spec.ts

```diff
@@ -19,6 +15,7 @@ test(__filename, async({ page }) => {
   await page.locator('html').press('Tab');
   await page.locator('html').press('Tab');
   await page.locator('html').press('Tab');
+  await page.locator('html').press('Tab');
```

Comment:
looks like the test was incorrect so I have changed to match the intentions as per comments